apps/vs-code-designer/src/app/utils/dotnet/getBrewPackageName.ts

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { dotnet, dotNetSDKMajorVersion } from '../../../constants';
+import { executeCommand } from '../funcCoreTools/cpUtils';
+
+/**
+ * Gets .NET SDK brew package name.
+ * @returns {string} Returns full package name for brew.
+ */
+export function getBrewPackageName(): string {
+  return `${dotnet}@${dotNetSDKMajorVersion}`;
+}
+
+/**
+ * Gets installed dot net sdk brew package.
+ * @returns {Promise<string | undefined>} Returns installed full package name for brew.
+ */
+const brewPackageName: string = getBrewPackageName();
+export async function tryGetInstalledBrewPackageName(): Promise<string | undefined> {
```

Comment:
Can you make this dynamic? Not stick exactly with brewPackageName of the dotnet, and send the name as a param